pubspec.yaml

```
@@ -15,7 +15,7 @@ dependencies:
   fixnum: ^1.0.0
   googleapis_auth: ^1.1.0
   meta: ^1.3.0
-  http: ^0.13.0
+  http: ^1.0.0
```

We want a wide range here to support migration!

```suggestion
  http: '>=0.13.0 <2.0.0'
```